sugoi-api-rest-services/src/main/java/fr/insee/sugoi/services/services/PermissionService.java

```diff
@@ -59,7 +61,12 @@ public boolean isPasswordManager(String realm, String userStorage) {
 
   public boolean isApplicationManager(String realm, String userStorage, String application) {
     List<String> searchRoleList =
-        getSearchRoleList(realm, userStorage, application, passwordManagerRoleList);
+        getSearchRoleList(realm, userStorage, application, applicationManagerRoleList);
+    return checkIfUserGetRoles(searchRoleList);
+  }
+
+  public boolean isApplicationManager(String realm) {
```

Comment:
maybe isApplication**s**Manager or something like that to understand that it has rights on all application on all userstorage ? (if it's what is done here)

Reply:
No this methods check if the user is applicationManager for one app in the realm.